index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <nav id="navbar" style="">
+        <header>Entertainment</header>
+        <a href="#ZELDA" class="nav-link">ZELDA</a>
+        <a href="#SMASH_BROSS" class="nav-link">SMASH BROSS</a>
+        <a href="#SPIDER_MAN" class="nav-link" >SPIDER-MAN</a>
+        <a href="#FORTNITE" class="nav-link">FORTNITE</a>
+        <a href="#THE_WEEKND" class="nav-link">THE WEEKND</a>
+        <a href="#" class="main-section"></a>
+    </nav>
+
+    <main id="main-doc">
+       
+        <section class="main-section" id="ZELDA">
+            <header>ZELDA</header>
+            <P><code>The Legend of Zelda</code> is a high fantasy action-adventure video game franchise created by Japanese game designers Shigeru Miyamoto and Takashi Tezuka. It is primarily developed and published by Nintendo, although some portable installments and re-releases have been outsourced to Capcom, Vanpool, and Grezzo. The gameplay incorporates action-adventure and elements of action RPG games.</P>
+            <p>he series centers on the various incarnations of Link, a courageous young Hylian man, with pointy elf-like ears; and Princess Zelda, a magical princess that is the mortal reincarnation of the goddess Hylia; as they fight to save the magical land of Hyrule from Ganon, an evil warlord turned demon king, who is the principal antagonist of the series. Ganon wishes to use the Triforce, a sacred relic left behind by the three goddesses that created Hyrule to remake the world in his own dark image. When gathered together, the power of the Triforce can grant any wish its user desires; however, if someone with a heart that does not possess a balance of the three virtues of Power, Courage and Wisdom attempts to touch the Triforce, it will split into three triangles and bond with three people whose hearts embody the required virtue. </p>
+            <p>Some of the games you can play are:</p>
+            <li>The Legend of Zelda (1986)</li> 
+            <li>Zelda II: The Adventure of Link (1987)</li>
+            <li>The Legend of Zelda: A Link to the Past (1991)</li>
+            <li>The Legend of Zelda: Link's Awakening (1993)</li>
+            <li>The Legend of Zelda: Ocarina of Time (1998)</li>
+            <li>The Legend of Zelda: Majora's Mask (2000)</li>
+        </section>
+        <section class="main-section" id="SMASH_BROSS">
```

Comment:
Please consider changing all of the section areads and using div tag instead, since this is more semantic and resalst on the linter guidelines.